test/config/install_coreos_maximum_QuantaT41.json

```diff
@@ -0,0 +1,51 @@
+{
+    "bootstrap-payload": {
+        "name": "Graph.InstallCoreOS",
+        "options": {
+            "defaults": {
+                "hostname": "rackhdnode",
```

Comment:
all your other max payloads have a user of 'rackhd-node'.  Wasn't sure if you're tying to follow a pattern here?